curl-and-grep/curl-and-grep.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: tekton.dev/v1alpha1
+kind: Task
+metadata:
+  name: curl-and-grep
+spec:
+  inputs:
+    params:
+    - name: url
+      description: URL to be curl'ed.
+    - name: text
+      description: text to grep
+      default: "HTTP"
```

Comment:
Should this be a default or should we make this parameter required (aka no default value)

Reply:
The motivation is to allow this task to be usable even in the case if there is no expected text in the body. So that, we can parse headers, where is "HTTP" should exist anyway. 
Initially, I was thinking to create separate tasks for curl and for grep, but one combined task could be better, to decrease the number of steps in pipelines and make them easier to use.

Reply:
NIT: you may want to add a comment about this in the docs